app/lib/frontend/handlers/pubapi.dart

```diff
@@ -55,6 +44,17 @@ class PubApi {
       await packageBackend.pubServer
           .download(_replaceHost(request.requestedUri), package, version);
 
+  /// Getting information about a specific (package, version) pair.
+  /// https://github.com/dart-lang/pub/blob/master/doc/repository-spec-v2.md#deprecated-inspect-a-specific-version-of-a-package
+  @EndPoint.get('/api/packages/<package>/versions/<version>')
```

Comment:
We need a comment explaining why this ordering is important..

Reply:
IMO, this can be a `// HACK: ....` explanation, also please reference the issue.

Reply:
IMO, this hack is acceptable because it only affects package versions that are suffixed `-XXX.tar,gz`

And the API end-point affected is deprecated.